supplement.csv

```diff
@@ -1,7 +1,88 @@
+半旗,/,noun,1911,"At half-mast, 半旗, 半桿旗",衛禮賢『德英華文科學字典』s.v. Halbmast,
+伴侶,/,noun,1848,"COMPANION, 伴侶, 匹耦, 匹配, 儔侶, 羣朋, 伙伴, 伙計, 夥計, 交遊, 同僚",麥都思『英華字典』s.v. Companion+,
+幫,/,noun,1872,"Party or company, 羣黨, 朋黨, 班, 幫",盧公明『英華萃林韻府』s.v. Party,
+綁票,/,verb,1917,"Kidnap, 綁票, 請財神","A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",
+保留,1,verb,1908,"To hold or retain, 保有, 保留",顏惠慶『英華大辭典』s.v. Keep+,
+保險公司,/,noun,1869,"Insurance company, 保險會, 燕梳會, 擔保會, 保險公司",羅存德『英華字典』s.v. Insurance+,
+保證書,/,noun,1881,?,黃遵憲『日本國志』刑法志二,
+報告,?,noun,1881,?,黃遵憲『日本國志』刑法志一,
+暴風,/,noun,1822,"TEMPEST of wind, 暴風",R. Morrison『英華字典』s.v. Tempest+,
+暴力,/,noun,1908,"Power exerted against will or consent, 抗願之力, 逼迫之力, 壓力, 暴力, 暴行",顏惠慶『英華大辭典』s.v. Force+,
+暴雨,/,noun,1857,有燥土而時有暴雨、成大水不能潤土者,"W. Muirhead in『六合叢談』1/13, 10",
+北極星,1,proper noun,1815,"A constellation, 北極星","R. Morrison, _A Dictionary of the Chinese Language in three parts_",
+備考,/,verb,1865,"備考, to retain for future investigation,—said of literary subjects",R. Morrison『五車韻府』s.v. 𤰇/備/僃/偹,
+背景,1+2,noun,1917,"背景 … background in theatricals, also used figuratively, a sequel","A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",
+背面,/,noun,1822,"Inside or back surface, 背面",R. Morrison『英華字典』s.v. Outside,
+被告,/,noun,1703,?,"F. Varo, _Arte de la lengua Mandarina_",Lemma not found using OCR
+被面,/,noun,1848,"COVERLET, 被窩, 被面, 被禪, 裨臂, 蓋被, 衾裯, 紟, 袣, 芾, 幭, 披單, 單紟",麥都思『英華字典』s.v. Coverlet,
+被殺,/,preposition + verb,1857,又辨金色之吏、曰穵林敦、亦被殺,"『六合叢談』1/4, 15+",
+本能,/,noun,1636,各独難普本能、彼此互合、乃易顕舌,陽瑪諾『聖經直解』,
+本體,/,noun,1844,"Essence, 精, 本體, 精氣",衛三畏『英華韵府歷階』s.v. Essence,
+本文,original text,noun,1857,合衆領事、及日本官、於兩國方言文字、均不能互相明曉、故於合約中准用荷蘭字為本文,"『六合叢談』1/12, 11",
+本土,local,noun,1857,有本土邏卒四人、為華民所殺,"『六合叢談』1/4, 15+",
+本義,/,noun,1869,"The proper sense of a word, 本意, 本義",羅存德『英華字典』s.v. Proper,
+本質,/,noun,1672,夫風之本質乃地所発乾熱之気有多端可証一試,南懐仁『坤輿圖説』,
+鼻音,/,noun,1869,"Ditto or nasal sound, 鼻音, 鼻聲",羅存德『英華字典』s.v. Twang+,
+比喻,/,noun,1703,"The conditional is rendered by the following particles: 若是, 若, 假如, 如, 譬如, 比喻, 比方","F. Varo, _Arte de la lengua Mandarina_ p. 111+", quote taken from 2000 English translation by W. S. Coblin and J. A. Levi
+比重,1,noun,1869,"Specific gravity, 比重",羅存德『英華字典』s.v. Gravity+,
+筆名,/,noun,1908,"A false name, 偽名, 冒名, 假名, 別名, 筆名",顏惠慶『英華大辭典』s.v. Pseudonym,
+必然,/,adverb,1703,?,"F. Varo, _Arte de la lengua Mandarina_",lemma not found using OCR
+變態,1,noun,1885,常恐後人體，變態猶未盡,黃遵憲『雜感·其三』,
+便秘,/,noun,1844,"CONSTIPATION, 大便秘, 閉恭",W. Williams『英華韵府歷階』s.v. Constipation,
+標本,/,noun,1902,?,梁啓超『飲冰室合集』,
+標記,/,noun,1858,朔望上下弦表、歷年測量寒熱表、行進江海標記、潮汐表、中外寄書表,"『六合叢談』2/1, 15",?
+標準語,/,noun,1916,"~ language, 標準語",赫美玲『官話』s.v. Standard,
+表面張力,/,noun,1911,"Surface tension, 表面張力",衛禮賢『德英華文科學字典』s.v. Oberflächenspannung,
+表情,/,noun,1908,"Expression of the passions, 表情, 情徵",顏惠慶『英華大辭典』s.v. Pathognomy,
+冰山,/,noun,1602,?,利瑪竇『坤輿萬國全圖』,
+冰箱,1,noun,1869,"Refrigerator, 涼箱, 冰箱",羅存德『英華字典』s.v. Refrigerator,
+兵營,/,noun,1922,"CAMP, 營, 兵營, 營寨",R. Morrison『英華字典』s.v. Camp+,
```

Comment:
```suggestion
兵工廠,/,noun,1917 (mid-1916),"Arsenal 軍械局, 兵工廠","A. H. Mateer, _Hand Book of New Terms and Newspaper Chinese_",
兵營,/,noun,1922,"CAMP, 營, 兵營, 營寨",R. Morrison『英華字典』s.v. Camp+,
```
Be extra careful in these cases. Mateer’s cut-off date was mid-1916 according to herself, and Chu is known to be penning his in 1917. We do want to establish the exact priority here.

No comma in the original.